src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+serverName=ec2-54-87-34-201.compute-1.amazonaws.com
+databaseName=ddptlc1075meqg
+pgPort=5432
+user=qfyyvuhtqcqucn
+password=097d7d6c14507f683e6ed6753549d2e9100b6a734c8c5d8a238e611ea374b0f9
```

Comment:
@AlexKievDeveloper  Please remove credentials from repo. 
Do you already deploy your code to env? 
Can you get credentials from env?

Reply:
> @AlexKievDeveloper Please remove credentials from repo.
> Do you already deploy your code to env?
> Can you get credentials from env?

Ok. 
Not yet. 
Yes I can